src/Components/test/E2ETest/ServerExecutionTests/ComponentHubReliabilityTest.cs

```diff
@@ -118,6 +119,7 @@ public async Task StartCircuitCausesInitializationError()
         }
 
         [Fact]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/29326")]
```

Comment:
This is redundant and you've closed the referenced issue. Suggest removing.

Reply:
```suggestion
```